### DISCUSSION TITLE
there is a prompt in chrome of initHighlightingOnLoad

### DISCUSSION THREAD

**Discussion by hiproz** [DirectoryLister/DirectoryLister#571] [General]
i tested the 3.7.5 release in php8.0 of centos, there was a prompt in console:

"Deprecated as of 10.6.0. initHighlightingOnLoad() is deprecated.  Use highlightAll() instead."

i used highlightAll for testing, the app worked fine. so maybe it should be replace?

the source code is app.js line 51:

hljs.initHighlightingOnLoad();

**Comment by PHLAK** [comment]
Thanks for catching this. Will update this soon.
